### ISSUE TITLE
Clapper as an UPnP DLNA player

### ISSUE THREAD

**Issue by juxuanu** [Rafostar/clapper#437]
I have Kodi set up on my repurposed-as-a-TV computer. Kodi supports advertising itself as an UPnP DLNA server (which Avahi daemon sees and VLC can play from correctly). See https://kodi.wiki/view/Settings/Services/UPnP_DLNA.
Right now, with the available Gnome applications, it's a bit hard to easily play content from such sources without resorting to VLC or `djmount` [[link]](https://djmount.sourceforge.net/). 
So, I wonder if this would be something that Clapper could purse. As a first thought, I'd imagine a new button on the initial screen, "Browse local network for available media servers...", etc. I'm guessing it could integrate with Avahi, or whatever. I understand it'd be a big new chunk of code and I don't know if it belongs to a video player to be able to browse the available content, but given I don't see an easy way at the moment, I thought it'd be cool. What do you think? Would this make sense?

**Comment by Rafostar** [comment]
While I agree this is a good use-case for a video player, I am not sure if I want to make this part of the "core" Clapper application itself, considering how much new views/windows + dependencies this would need.

This might be something that will end up better way in a dedicated app for such usage with Clapper implemented as an embedded player in that app. But who knows? Let's leave it as an open feature request for the time being.

**Comment by juxuanu** [comment]
Yeah, it'd might be better off in its own app. Given you created the Clapper GTK library, integration would be trivial.

**Comment by Rafostar** [comment]
While not KODI, but someone already managed to integrate Clapper into an Emby client app:
https://github.com/tsukinaha/tsukimi

This is a nice example that an app that wants to use Clapper for some very specific usage can be better then trying to force new functionality into core Clapper app itself.
